packages/cloud-cognitive/src/components/CreateTearsheet/CreateTearsheetStep.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright IBM Corp. 2021, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { forwardRef } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { pkg } from '../../settings';
+import { CREATE_TEARSHEET_STEP } from './constants';
+
+const componentName = 'CreateTearsheetStep';
+const blockClass = `${pkg.prefix}--tearsheet__step`;
```

Comment:
Should the block class not reference 'create tearsheet'?

Reply:
Does this component need to accept additional props?

Reply:
I don't think it need additional props, this is simply a wrapper component for each step. It does have props of it's own but not likely that it needs additional. And yes, I agree the block class should reflect the create tearsheet block class, something like this `${pkg.prefix}--tearsheet-create__step`